python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -257,18 +261,41 @@ def refresh_progress_bar(self, resource_manager: ResourceManager) -> None:
             self.progress_bar.refresh()
 
     def summary_str(self, resource_manager: ResourceManager) -> str:
-        queued = self.num_queued() + self.op.internal_queue_size()
+        # Active tasks
         active = self.op.num_active_tasks()
-        desc = f"- {self.op.name}: {active} active, {queued} queued"
+        desc = f"- {self.op.name}: Tasks: {active}"
         if (
             self.op._in_task_submission_backpressure
             or self.op._in_task_output_backpressure
         ):
-            desc += " 🚧"
-        desc += f", [{resource_manager.get_op_usage_str(self.op)}]"
+            desc += " [backpressured]"
+
+        # Active/pending actors
+        active = self.op.num_active_actors()
+        pending = self.op.num_pending_actors()
+        if active or pending:
+            assert isinstance(self.op, ActorPoolMapOperator)
+            actor_str = f"; Actors: {active}"
+            if pending > 0:
+                actor_str += f", (pending: {pending})"
+            if self.op._actor_locality_enabled:
+                actor_str += " " + locality_string(
+                    self._actor_pool._locality_hits, self._actor_pool._locality_misses
```

Comment:
BTW @raulchen i still had to leave some `ActorPoolMapOperator` specific logic, in order to add the locality string (checking the attribute `if self.op._actor_locality_enabled` which only exists for `ActorPoolMapOperator`). but i just do this with the `assert isinstance(self.op, ActorPoolMapOperator)` check to satisfy type checking. let me know if you prefer another approach

Reply:
oh, I think we can still keep the locality info at the end (still reported by `progress_str`), because it's operator-specific